pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -629,13 +651,75 @@ func (b *OFBridge) RetryInterval() time.Duration {
 	return b.retryInterval
 }
 
+func (b *OFBridge) queryTableFeatures() {
+	mpartRequest := &openflow13.MultipartRequest{
+		Header: openflow13.NewOfp13Header(),
+		Type:   openflow13.MultipartType_TableFeatures,
+		Flags:  0,
+	}
+	mpartRequest.Header.Type = openflow13.Type_MultiPartRequest
+	mpartRequest.Header.Length = mpartRequest.Len()
+	// Use a buffer for the channel to avoid blocking the OpenFlow connection inbound channel, since it takes time when
+	// sending the Multipart Request messages to modify the tables' names. The buffer size "20" is the observed number
+	// of the Multipart Reply messages sent from OVS.
+	tableFeatureCh := make(chan *openflow13.MultipartReply, 20)
```

Comment:
I think it's good enough, but I assume it depends on the number of tables? I feel like we don't need to be conservative here, since it's not like we are using up a lot of memory with a buffered channel of pointers. So we could have picked something like 100.